docs/_data/navigation.yml

```
@@ -0,0 +1,35 @@
+rse:
+  - title: New to RSE?
+    children:
+    - title: "Information for new RSEs"
+      url: /roles/new-rse/#resources-and-information-for-new-rses
+    - title: "What is an RSE"
```

Very minor but suggesting the addion of a question mark here:

```suggestion
- title: "What is an RSE?"
```